### PR TITLE
add justbuild package

### PR DESCRIPTION
**Pull request by asartori86**

### CODE REVIEW

File: var/spack/repos/builtin/packages/justbuild/package.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+
+class Justbuild(Package):
+    "just, a generic build tool"
+
+    git = "https://github.com/just-buildsystem/justbuild.git"
+
+    tags = ["build-tools"]
+
+    executables = ["^just$", "^just-mr$"]
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/justbuild/package.py]: Are both of these always installed? Only one (that is always installed) is needed for https://spack.readthedocs.io/en/latest/packaging_guide.html#making-a-package-discoverable-with-spack-external-find.

Comment (tldahlgren) [reply var/spack/repos/builtin/packages/justbuild/package.py]: If your goal is to make sure the executables are installed, you could add sanity checks (see https://spack.readthedocs.io/en/latest/packaging_guide.html#adding-sanity-checks).

**Review by tldahlgren — COMMENTED** [review]
Let me know if you want to make any changes to this package in this PR.

File: var/spack/repos/builtin/packages/justbuild/package.py (hunk 1 above)
Comment (asartori86) [reply var/spack/repos/builtin/packages/justbuild/package.py]: thanks @tldahlgren 

I implemented your suggestions. Would you mind to take a second look at it?

both the executables are always installed

**Review by tldahlgren — APPROVED** [review]

**Pull request merged**